example/androidlib/java/5-R8/build.mill

```diff
@@ -75,7 +69,7 @@ object app extends AndroidR8AppModule { // <2>
       false
     }
 
-    override def androidReleaseSettings: T[AndroidBuildTypeSettings] = Task.Input {
+    override def androidReleaseSettings: T[AndroidBuildTypeSettings] = Task {
```

Comment:
This change will result ins missing change detection for `test-proguard-rules.pro` file.

Best, it so make that file a 

```scala
def testProgurardRulesPro = Task.Source("test-proguard-rules.pro")
```

and use it below.

Reply:
Same in the other locations.

Reply:
thanks @lefou , I suggested to @souvlakias to split this PR into two this morning, do the proguard changes for hilt and providers first, I'd like to experiment a bit with this to find a better solution, as at first I made a direct translation from gradle -> mill